### PR TITLE
smoke b2c payment method update

### PR DESCRIPTION
**Pull request by khushal-rathore-metacube**
https://spryker.atlassian.net/browse/CC-21836

Performed smoke b2c payment method update

- Update payment method → Check in Yves
- Deactivate payment method → Check in Yves
- Unset payment method for store → Check in Yves

### CODE REVIEW

File: resources/steps/pdp_steps.robot [hunk 1]
```diff
@@ -177,4 +177,5 @@ Yves: merchant is (not) displaying in Sold By section of PDP:
     Wait Until Element Is Visible    ${pdp_product_sku}[${env}]
     Try reloading page until element is/not appear:    xpath=//section[@data-qa='component product-configurator']//div[contains(text(),'${merchantName}')]     ${condition}    20    5s  
    
-    
\ No newline at end of file
+Yves: add a concrete product to cart
```
Comment (AntonShopin) [inline resources/steps/pdp_steps.robot:180]: why do we need this new keyword? 
what is wrong with `Yves: add product to the shopping cart`?

File: tests/smoke/smoke_b2c.robot [hunk 2]
```diff
@@ -599,3 +600,22 @@ Refunds
...
+    Yves: go to PDP of the product with sku:    020
+    Yves: add a concrete product to cart
+    Yves: go to b2c shopping cart    
+    Yves: click on the 'Checkout' button in the shopping cart
+    Yves: fill in the following new shipping address:
+    ...    || firstName               |     lastName                    |    street        |    houseNumber      |    city          |    postCode    |    phone        ||
+    ...     || ${yves_second_user_first_name}    |     ${yves_second_user_last_name}         |    ${random}              |    ${random}              |    Berlin${random}        |   ${random}              |    ${random}       ||
+    Yves: submit form on the checkout
+    Yves: select the following shipping method on the checkout and go next:     Standard: €4.90
+    Page Should Not Contain Element    ${checkout_payment_invoice}
```
Comment (AntonShopin) [inline tests/smoke/smoke_b2c.robot:618]: Please don't use 'default' robot keywords in tests. Create a new keyword (if it doesn't exist) to check if the payment method is displayed or not and the expected condition for it

File: tests/smoke/smoke_b2c.robot [hunk 3]
```diff
@@ -599,3 +600,22 @@ Refunds
     [Teardown]    Run keywords    Zed: login on Zed with provided credentials:    ${zed_admin_email}
     ...    AND    Zed: activate following discounts from Overview page:    Tu & Wed $5 off 5 or more    10% off $100+    20% off cameras    Tu & Wed €5 off 5 or more    10% off minimum order
  
+Payment_method_update
```
Comment (AntonShopin) [inline tests/smoke/smoke_b2c.robot:603]: you need to change the order of your steps. 

1. Without any changes in Zed -> check that the Invoice payment method is displayed during the checkout 
2. Edit in Zed (deactivate)
3. Return to Yves checkout and see that now it's not displayed 
4. Activate it back in Zed so we return to the initial state

**Review by AntonShopin — CHANGES_REQUESTED** [review]

File: tests/smoke/smoke_b2c.robot (hunk 3 above)
Comment (khushal-rathore-metacube) [reply tests/smoke/smoke_b2c.robot:603]: Done

File: tests/smoke/smoke_b2c.robot (hunk 2 above)
Comment (khushal-rathore-metacube) [reply tests/smoke/smoke_b2c.robot:618]: Done

File: resources/steps/pdp_steps.robot (hunk 1 above)
Comment (khushal-rathore-metacube) [reply resources/steps/pdp_steps.robot:180]: Done

File: resources/steps/checkout_steps.robot [hunk 4]
```diff
@@ -204,5 +204,10 @@ Yves: assert merchant of product in cart or list:
...
 
-
-
+Yves: check payment method presence in checkout process:
+    [Arguments]    ${payment_method_locator}    ${condition}
+    IF    '${condition}' == 'True'
+        Page Should Contain Element    ${payment_method_locator}
+    ELSE IF    '${condition}' == 'False'
+        Page Should not Contain Element    ${payment_method_locator}   
+    END
```
Comment (AntonShopin) [inline resources/steps/checkout_steps.robot:213]: 1. Please rename it. Maybe: `Yves: check that the payment method is/not present in the checkout process:` ?
2. Avoid using Upper case in boolean values. it should be true or false (lower case)

Comment (khushal-rathore-metacube) [reply resources/steps/checkout_steps.robot:213]: fixed

**Review by AntonShopin — APPROVED** [review]

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**